ansible/roles/test/files/ptftests/py3/IP_decap_test.py

```diff
@@ -440,8 +441,39 @@ def send_and_verify(self, dst_ip, exp_port_lists, src_port, dut_index, outer_pkt
                              exp_ttl,
                              str(expected_ports)))
 
-        matched, received = verify_packet_any_port(
-            self, masked_exp_pkt, expected_ports, timeout=1)
+        try:
+            matched, received = verify_packet_any_port(
```

Comment:
Why not cover the first send in the for loop? 
And the concern for this change is it may hide some real issue.